css/style.css

```diff
@@ -46,14 +46,17 @@ Developer @GA in NYC
 /*******************************************/
 
 body {
-     color: #000;
-     font-size: 12px;
+         font-size: 12px;
```

Comment:
Try to be consistent with your indentation. Ideally you want everything to line up.